test/_features/step_definitions/steps.js

```diff
@@ -73,6 +73,12 @@ Then('I should be on the {string} page showing {string}', async function (uri, h
   expect(await this.page.innerText('body')).to.include(heading);
 }.bind(World));
 
+Then('I should be redirected to the collection page showing {string}', async function (heading) {
+  await this.page.waitForSelector('body', { timeout: 15000 });
+  expect(await this.page.url()).to.include('https://www.biometric-residence-permit.service.gov.uk/collection/where');
```

Comment:
Step definitions should be reusable where possible.

check out:
`expect(new URL(await this.page.url()).pathname).to.eql(`${this.subApp}/${uri}`);`
from the test above
 
you may need to alter change the value of subapp to a value you can pass in
check out the first "given" where its set:"

`Given('I start the {string} application journey', async function (subApp) {
  this.subApp = subApp === 'base' ? '' : `/${subApp}`;
  await this.page.goto(`${domain}${this.subApp}`);
}.bind(World));
`